portfolio/src/main/webapp/script.js

```diff
@@ -260,21 +298,55 @@ function makeCommentAuthorElement(author) {
 }
 
 /**
- * Inserts new comment into the comment Display
+ * Gets form input and uploads it to the servers
  */
-function updateComments() {
+function updateComments() { 
+  //retrieve form values
   var commentText = clearFormValue("inputComment");
   var commentAuthor = clearFormValue("inputName");
-  if (validComment(commentText)) {
-    const params = new URLSearchParams();
-    params.append('comment-text', commentText);
-    params.append('comment-author', commentAuthor);
-    fetch('new-comment', {method: 'POST', body: params}).then(refreshComments);
+
+  //TODO: replace loading map and comments with "loading..."
+  if (!validComment(commentText)) {
+    alert("invalid comment");
+    return;
   }
+
+  const latLng = tempMarker? tempMarker.getPosition(): DEFAULT_COORDS;
+  console.log(latLng.toString());
```

Comment:
console.log should only be used while debugging.

Reply:
Yes, I meant to remove them before submitting so I'll delete them in the next commit.